udagram-api-user/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:13
```

Comment:
You should use a more defined version, and leverage alpine https://hub.docker.com/_/node/tags?page=1&name=alpine